lib/product.rb

```diff
@@ -0,0 +1,19 @@
+class Product
+  attr_reader :category,
+              :name,
+              :unit_price,
+              :quantity
+
+
+  def initialize(category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @quantity = quantity.to_i
+    @unit_price = unit_price
+  end
+
+  def total_price
+    total_cost = (@unit_price * @quantity.to_i)
```

Comment:
This is correct! but we don't need the variable assignment to `total_cost` as total price should just return the value of the calculation